.travis.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     include:
         -
             stage: Static Code Analysis
-            php: 7.3
+            php: '7.3'
```

Comment:
please revert

Reply:
why? PHP version is not a float value, but a string. PHP 7.0 is not `7.0`, but `7` on the [list](https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/builds/613530218).

Reply:
i don't see anyhow this argument.
even if string, then `php: 7.4snapshot` and not `php: '7.4snapshot'`.
same, the `stage` is a string, but we don't use quotes. they are not needed

Reply:
`stage` is a string because it is a string.

I was always thinking we want be explicit over implicit - I guess not in this case.

Reply:
i see no reasoning why `stage` should not have quotes and `php` should
we have, on master line:
```
stage: Static Code Analysis
php: 7.4snapshot
```

both are strings. none is using quotes